### PR TITLE
Update cosmic_settings.ftl PL translation

### PR DESCRIPTION
**Pull request by VandaLHJ**
Added keyboard shortcuts and minor changes

### CODE REVIEW

File: i18n/pl/cosmic_settings.ftl [hunk 1]
```diff
@@ -123,6 +123,16 @@ orientation = Kierunek
 scheduling = Harmonogram
     .manual = Ręcznie ustawiony harmonogram
 
+dialog = Dialog
+    .title = Czy Zachować Te Ustawienia Wyświetlacza?
+    .keep-changes = Zachowaj Zmiany
+    .change-prompt = Ustawienia automatycznie powrócą do poprzednich za { $time ->
+        [1] sekundę.
```
Comment (xalt7x) [inline i18n/pl/cosmic_settings.ftl:130]: @VandaLHJ ,
This improvement with plural forms doesn't work on my system. For some reason, after this change, the entire line is displayed in English...

File: i18n/pl/cosmic_settings.ftl [hunk 2]
```diff
@@ -123,6 +123,16 @@ orientation = Kierunek
...
+        [few] {$time} sekundy.
+        [other] {$time} sekund.
+    }
```
Comment (xalt7x) [inline i18n/pl/cosmic_settings.ftl:133]: @VandaLHJ ,
This improvement with plural forms doesn't work on my system. For some reason, after this change, the entire line is displayed in English...

Comment (xalt7x) [reply i18n/pl/cosmic_settings.ftl:133]: I've got it working with this:
```
    .change-prompt = Ustawienia automatycznie powrócą do poprzednich za { $time } { $time ->
        [1] sekundę.
        [few] sekundy.
        *[other] sekund.
    }
```
It seems that an asterisk is mandatory.

зображення

Comment (VandaLHJ) [reply i18n/pl/cosmic_settings.ftl:133]: Yep, you are right, my mistake. Added it.

**Review by wash2 — APPROVED** [review]

**Pull request merged**